.github/workflows/release-code-freeze.yml

```diff
@@ -77,15 +77,15 @@ jobs:
       releaseVersion: ${{ needs.check-code-freeze.outputs.nextReleaseVersion }}
     secrets: inherit
 
-  slack-notification:
-    name: "Send notification to Slack"
+  slack-notification-pr-comment:
```

Comment:
We can revert to the original name `slack-notification`

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/5586/commits/62038c903bca224966cac8a4f64d2afa6f546a7c